### DISCUSSION TITLE
Max-width best practice ?

### DISCUSSION THREAD

**Discussion by KatyPurry** [biati-digital/glightbox#487] [Q&A]
Hello,
I'm using the V4 on my wordpress site, it works like a charm. By default on media > 769px it displays at 100% : 

```
@media (min-width: 769px) {
  .gslide-image img {
    max-height: 97vh;
    max-width: 100%;
  }
}
```

If I set to 90% of the max-width, can it break things somewhere else or is it good?

Thank you !

**Comment by gingerchew** [comment]
v4 is still in beta, so there isn't a "best practice" so to say. I would say just go for it 😄
